6_visualize/src/create_gif_tasks.R

```diff
@@ -35,7 +35,19 @@ create_gif_tasks <- function(timestep_ind, folders, storm_cfg){
     },
     command = function(task_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
-      sprintf("prep_storm_point_fun('2_process/out/storm_points_interp.rds.ind', I('%s'), hurricane_col)", cur_task$timestep) # pass in storm_points.ind as arg to create_gif_tasks??
+      sprintf("prep_storm_point_fun(storm_points_sf, I('%s'), hurricane_col)", cur_task$timestep)
+    }
+  )
+
+  precip_frame <- scipiper::create_task_step( # not sure why this is called "frame".
```

Comment:
Each task builds a single animation frame illustrating the state at a single timestep. Each step within the task (e.g., one call to the step outlined by `precip_frame`) adds a layer (e.g., precip) to that animation frame.

Reply:
However, I also agree that "frame" is maybe misleading given that precip is just one layer within the frame. Maybe "precip_layer" would be more informative? or maybe just "precip" would be sufficient?